.github/workflows/ci.yml

```diff
@@ -43,7 +43,7 @@ jobs:
           cargo --version
 
       - name: Install cargo crates
-        run: cargo install cargo-binutils rustfilt
+        run: cargo install cargo-binutils@0.3.6 rustfilt@0.2.1
```

Comment:
https://github.com/anholt/kerla/actions/runs/10983975544/job/30494006359#step:6:63

```
Caused by:
  package `regex-automata v0.4.7` cannot be built because it requires rustc 1.65 or newer, while the currently active rustc version is 1.61.0-nightly
error: failed to compile `rustfilt v0.2.1`, intermediate artifacts can be found at `/tmp/cargo-install2m2JTq`

Caused by:
  package `regex-syntax v0.8.4` cannot be built because it requires rustc 1.65 or newer, while the currently active rustc version is 1.61.0-nightly
     Summary Failed to install cargo-binutils, rustfilt (see error(s) above).
```

Reply:
Makes sense. Thanks!